Dockerfile

```diff
@@ -69,6 +69,10 @@ COPY docker-assets/initialize-httpd-auth.sh   /usr/bin
 
 COPY docker-assets/initialize-httpd-auth.service /usr/lib/systemd/system/initialize-httpd-auth.service
 
+## Make sure sssd has the right startup conditions
+RUN  mkdir -p /etc/systemd/system/sssd.service.d
+COPY docker-assets/sssd-startup.conf      /etc/systemd/system/sssd.service.d/startup.conf
```

Comment:
Why the extra spacing here?

Reply:
sorry, was aligned with the lower COPY, will update.